pkg/operator/sync.go

```diff
@@ -927,6 +928,65 @@ func (optr *Operator) safetySyncControllerConfig(config *renderConfig) error {
 // the operator version and controller version match. We can call it from safetySyncControllerConfig
 // because safetySyncControllerConfig ensures that the operator and controller versions match before it syncs.
 func (optr *Operator) syncControllerConfig(config *renderConfig) error {
+	editCCAnno := false
+	kubeConfigData, err := optr.clusterCmLister.ConfigMaps("openshift-config-managed").Get("kube-apiserver-server-ca")
```

Comment:
This section should most of the time should be a relative no-op. Should we move near the end of the sync?

Reply:
good idea. This will avoid us missing out on actually syncing the controller config

Reply:
I can move it down past part of the function but not too far since we need to add the annotation to the controllerConfig